band_together/project.godot

```diff
@@ -69,7 +69,7 @@ project/assembly_name="Band Together"
 
 [editor_plugins]
 
-enabled=PackedStringArray("res://addons/dialogic/plugin.cfg")
+enabled=PackedStringArray()
```

Comment:
I think this may be how  the dialogic plugin was removed? But you resolved this is in your most recent PR!